docs/sql-ref-syntax-aux-describe-table.md

```diff
@@ -118,9 +118,9 @@ to return the metadata pertaining to a partition or column respectively.
       "num_buckets": <num_buckets>,
       "bucket_columns": ["<col_name>"],
       "sort_columns": ["<col_name>"],
-      "created_time": "<timestamp_ISO-8601>",
+      "created_time": "<yyyy-MM-dd'T'HH:mm:ss.SSSXXX>",
```

Comment:
To double check, the example is `"created_time":"2020-04-07T14:05:43Z"`, which is correct?

Reply:
yep, I updated the docs to specify that we are converting to UTC (`...Z`)